docs/docs/using-the-template.md

```diff
@@ -0,0 +1,158 @@
+# Using the template
+
+Ok, so you've created your project. You've [read the opinions section](opinions.md). You're ready to start doing some work.
+
+Here's a quick guide of the kinds of things we do once our project is ready to go.
+
+## Set up version control
+
+Once you `cd` into the directory created by `ccds` you're ready to go.
+
+Often, we start by initializing a `git` repository to track the code we write in version control and collaborate with teammates. At the command line, you can do this with the following commands which do the following: turn the folder into a git repository, add all of the files and folders created by CCDS into source control (except for what is in the `.gitignore` file), and then make a commit to the repository.
+
+```bash
+git init
+git add .
+git commit -m "CCDS defaults"
+```
+
+We usually commit the entire default CCDS structure so it is easy to track in verison history the changes we make to the structure separately.
+
+Now that default layout is committed, you should push it to a shared repository. You can do this through the interface of whatever source control platform you use. This may be GitHub, GitLab, Bitbucket, or something else.
```

Comment:
```suggestion
Now that the default layout is committed, you should push it to a shared repository. You can do this through the interface of whatever source control platform you use. This may be GitHub, GitLab, Bitbucket, or something else.
```